app/views/monitoring.html

```diff
@@ -13,25 +13,27 @@
               </h1>
             </div>
             <alerts alerts="alerts"></alerts>
-            <div class="data-toolbar">
-              <ui-select ng-model="kindSelector.selected" theme="bootstrap" search-enabled="true" ng-disabled="kindSelector.disabled" title="Choose a resource" class="data-toolbar-dropdown">
-                <ui-select-match placeholder="Choose a resource">{{$select.selected.label ? $select.selected.label : ($select.selected.kind | humanizeKind : true)}}</ui-select-match>
-                <ui-select-choices repeat="kind in kinds | filter : {kind: $select.search} : matchKind | orderBy : 'kind'">
-                  <div ng-bind-html="(kind.label ? kind.label : (kind.kind | humanizeKind : true)) | highlight: $select.search"></div>
-                </ui-select-choices>
-              </ui-select>
-              <div class="vertical-divider"></div>
-              <div class="data-toolbar-filter form-inline">
-                <div class="form-group filter-controls">
-                  <label for="events-filter" class="sr-only">Filter by name</label>
-                  <input type="search"
-                        placeholder="Filter by name"
-                        class="form-control"
-                        id="events-filter"
-                        ng-model="filters.text">
+            <div fit class="data-toolbar" style="justify-content: space-between;">
```

Comment:
It's a little strange to set `justify-content` inline when nothing here in the markup makes it a flex container. It's only part the `data-toolbar` style. Maybe change this to `class="data-toolbar monitoring-toolbar"` and add the `justify-content` style to a `monitoring-toolbar` rule in `_data-toolbar.less`?